releasenotes/notes/retworkx-eol-bcbf18d805a73aed.yaml

```diff
@@ -0,0 +1,7 @@
+---
+deprecations:
```

Comment:
I'd probably make this an `upgrade` note too.

Reply:
Done.